deploy/cephfs/kubernetes/csi-provisioner-rbac.yaml

```diff
@@ -30,25 +27,25 @@ rules:
     verbs: ["get", "list", "watch"]
   - apiGroups: ["snapshot.storage.k8s.io"]
     resources: ["volumesnapshots"]
-    verbs: ["get", "list"]
+    verbs: ["get", "list", "watch"]
```

Comment:
why is this added? explain it in the commit message please

Reply:
not needed.. removed.